guides/common/modules/proc_installing-and-configuring-the-puppet-agent.adoc

```diff
@@ -13,6 +13,10 @@ endif::[]
 * The host must have a Puppet environment assigned to it.
 ifdef::satellite[]
 * The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
+The `puppet-agent rpm` is available within the following repositories:
+** {Team} {project-client-name} for RHEL 7 Server RPMs x86_64 - `{RepoRHEL7ServerSatelliteToolsProductVersion}`
+** {Team} {project-client-name} for RHEL 8 Server RPMs x86_64 - `{RepoRHEL8ServerSatelliteToolsProductVersion}`
+** {Team} {project-client-name} for RHEL 9 Server RPMs x86_64 - `{RepoRHEL9ServerSatelliteToolsProductVersion}`
```

Comment:
Reading through the rendered result, this is inconsistent. Copying for simplicity:
```
Red Hat Satellite Client 6 for RHEL 8 Server RPMs x86_64 - satellite-client-6-for-rhel-8-<arch>-rpms
Red Hat Satellite Client 6 for RHEL 9 Server RPMs x86_64 - satellite-client-6-for-rhel-9-<arch>-rpms
```

I think this would be better.

```suggestion
** {Team} {project-client-name} for RHEL 8 Server RPMs <arch> - `{RepoRHEL8ServerSatelliteToolsProductVersion}`
** {Team} {project-client-name} for RHEL 9 Server RPMs <arch> - `{RepoRHEL9ServerSatelliteToolsProductVersion}`
```